### PR TITLE
added export history

### PR DESCRIPTION
**Pull request by segler-alex**
Closes: #717

### CODE REVIEW

File: app/src/main/java/net/programmierecke/radiodroid2/ActivityMain.java [hunk 1]
```diff
@@ -678,9 +681,14 @@ public void onFileSelected(FileDialog dialog, File file) {
...
             FavouriteManager favouriteManager = radioDroidApp.getFavouriteManager();
+            HistoryManager historyManager = radioDroidApp.getHistoryManager();
 
             if (dialog instanceof SaveFileDialog) {
-                favouriteManager.SaveM3U(file.getParent(), file.getName());
+                if (selectedMenuItem == R.id.nav_item_starred) {
+                    favouriteManager.SaveM3U(file.getParent(), file.getName());
+                }else if (selectedMenuItem == R.id.nav_item_history) {
+                    historyManager.SaveM3U(file.getParent(), file.getName());
+                }
```
Comment (werman) [inline app/src/main/java/net/programmierecke/radiodroid2/ActivityMain.java]: But what about loading the history back?

Comment (segler-alex) [reply app/src/main/java/net/programmierecke/radiodroid2/ActivityMain.java]: i think history should not be loadable, it is history.

Comment (segler-alex) [reply app/src/main/java/net/programmierecke/radiodroid2/ActivityMain.java]: you can load it to add it to your favorites

Comment (werman) [reply app/src/main/java/net/programmierecke/radiodroid2/ActivityMain.java]: But why then export it =) At least in the issue user wanted to backup the history, which in my head means he would also want to restore it.

Comment (segler-alex) [reply app/src/main/java/net/programmierecke/radiodroid2/ActivityMain.java]: it is not for backup reasons in my opinion, it is to use it with different players.
why would you backup history? if you like a stream you add it to favorites, which you can backup.

Comment (werman) [reply app/src/main/java/net/programmierecke/radiodroid2/ActivityMain.java]: I agree, I too don't understand it. However in original issue user ask to be able to backup history for whatever reason...

Comment (segler-alex) [reply app/src/main/java/net/programmierecke/radiodroid2/ActivityMain.java]: i actually believe this is what they meant, but of course i could be wrong...

**Comment by segler-alex** [comment]
i merged this for now, a full backup needs another pull request

**Pull request merged**